app/components/createContainer/createContainerController.js

```diff
@@ -233,6 +234,185 @@ function ($q, $scope, $state, $stateParams, $filter, Container, ContainerHelper,
     return config;
   }
 
+  function confirmCreateContainer() {
+    var deferred = $q.defer();
+    Container.query({ all: 1, filters: {name: [$scope.config.name] }}).$promise
```

Comment:
We need to change this. This returns a list of containers containing `$scope.config.name` in their name and not containers with the exact name in my environment (see https://forums.docker.com/t/how-to-filter-docker-ps-by-exact-name/2880)

We need to add an extra check by looping in the container names.

Try to reproduce the following:

* Create a container named `some_test_container`
* Create a container named `test`
* The replace modal will be displayed